tests/backends/postgresql/tests.py

```diff
@@ -305,6 +305,12 @@ def test_connect_no_is_usable_checks(self):
         finally:
             new_connection.close()
 
+    def test_client_encoding(self):
+        if is_psycopg3:
+            self.assertEqual(connection.connection.info.encoding, "utf-8")
```

Comment:
Shouldn't this test (like the test above) copy the connection and change `connection_encoding` to let's say iso-8859 and verify that it is still utf-8?

Reply:
True, updated :+1: Thanks :gift: